lib/parser/builders/default.rb

```diff
@@ -437,6 +437,11 @@ def cvar(token)
         variable_map(token))
     end
 
+    def numparam(token)
+      n(:numparam, [ value(token)[1..-1].to_i ],
```

Comment:
I think you can use `value(token).drop(1)` here that should be equivalent but does not create the range object.

Reply:
I'd prefer to not use any destructive methods on values that I don't own unless there's a good reason for that. This string has at most 4 chars

Reply:
Its not destructive (and misnamed):

```
irb(main):001:0> a = [1,2,3]
=> [1, 2, 3]
irb(main):002:0> a.drop(1)
=> [2, 3]
irb(main):003:0> a
=> [1, 2, 3]
```

Reply:
That should be done by the lexer anyway.

Reply:
OMG forget the noise, this is a string, not an array.